sdk/platform/http_client/curl/src/az_curl.c

```diff
@@ -478,18 +476,23 @@ _az_http_client_curl_setup_url(CURL* p_curl, _az_http_request const* p_request)
   AZ_PRECONDITION_NOT_NULL(p_curl);
   AZ_PRECONDITION_NOT_NULL(p_request);
 
+  az_span request_url = { 0 };
+  // get request_url. It will have the size of what is has written in it only
```

Comment:
Not sure how to read this. Is this what you meant?
```suggestion
  // get request_url. It will have the size of what it has written in it only
```